fuzz/fuzz.go

```diff
@@ -0,0 +1,410 @@
+package fuzz
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"github.com/golang/mock/gomock"
+	"io"
+	"math"
+	"net/netip"
+	"testing"
+
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/compiler"
+	"github.com/brimdata/zed/compiler/data"
+	"github.com/brimdata/zed/compiler/optimizer"
+	"github.com/brimdata/zed/compiler/optimizer/demand"
+	"github.com/brimdata/zed/compiler/semantic"
+	"github.com/brimdata/zed/pkg/nano"
+	"github.com/brimdata/zed/pkg/storage/mock"
+	"github.com/brimdata/zed/runtime"
+	"github.com/brimdata/zed/zbuf"
+	"github.com/brimdata/zed/zcode"
+	"github.com/brimdata/zed/zio"
+	"github.com/brimdata/zed/zio/vngio"
+	"github.com/brimdata/zed/zio/zngio"
+	"github.com/brimdata/zed/zson"
+	"github.com/stretchr/testify/require"
+	"github.com/x448/float16"
+)
+
+func ReadZng(bs []byte) ([]zed.Value, error) {
```

Comment:
It's conventional to use [consistent case for initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).
```suggestion
func ReadZNG(bs []byte) ([]zed.Value, error) {
```